src/zjs_util.c

```diff
@@ -180,6 +177,41 @@ bool zjs_obj_get_int32(jerry_value_t obj, const char *name, int32_t *num)
     return true;
 }
 
+void zjs_copy_jstring(jerry_value_t jstr, char *buffer, jerry_size_t *maxlen)
+{
+    jerry_size_t size = jerry_get_string_size(jstr);
```

Comment:
Actually this is okay. The contract is, don't call this with NULL. That's specified in the function description. `@param buffer  A char * buffer with at least *maxlen bytes.`

This is the "stack" version so it should be called on a stack pointer that could never be NULL. So I think it's reasonable to leave this requirement on the caller.